### PR TITLE
[Owners] Removed Calibration APIs and related Enums from NI-DCPower

### PR DESCRIPTION
**Pull request by ni-pkhare**
### What does this Pull Request accomplish?
Removed calibration APIs and enums from metadata as well as generated files as it is decided not to support external calibration in gRPC.

The list of APIs decided to be removed is present in the below link.
https://nio365-my.sharepoint.com/:x:/g/personal/akshata_k_ni_com/EcTGOgHrnPhDrJCOJWMXe5sBMWJhiq-dItyT6nlvL5Xayg?e=UloYbf

Note: The changes are documented in CHANGES.md file

### Why should this Pull Request be merged?
Removed calibration APIs and enums

### What testing has been done?
Build successful

### CODE REVIEW

**Review by kruthi-NI — APPROVED** [review]

File: generated/nidcpower/nidcpower.proto [hunk 1]
```diff
@@ -104,7 +94,6 @@ service NiDCPower {
   rpc ConfigureVoltageLevelRange(ConfigureVoltageLevelRangeRequest) returns (ConfigureVoltageLevelRangeResponse);
   rpc ConfigureVoltageLimit(ConfigureVoltageLimitRequest) returns (ConfigureVoltageLimitResponse);
   rpc ConfigureVoltageLimitRange(ConfigureVoltageLimitRangeRequest) returns (ConfigureVoltageLimitRangeResponse);
-  rpc ConnectInternalReference(ConnectInternalReferenceRequest) returns (ConnectInternalReferenceResponse);
```
Comment (jonlugoatx) [inline generated/nidcpower/nidcpower.proto:107]: Is this one a cal api?

**Review by jonlugoatx — APPROVED** [review]

File: generated/nidcpower/nidcpower.proto (hunk 1 above)
Comment (Akshata440) [reply generated/nidcpower/nidcpower.proto:107]: Yes
image

File: source/codegen/metadata/nidcpower/CHANGES.md [hunk 2]
```diff
@@ -54,6 +52,29 @@ Following enums were defined :
...
+- CalibrationCloseAction
+
+## functions.py
+
+Following calibration functions were removed from the metadata :
+- ChangeExtCalPassword
+- GetCalUserDefinedInfoMaxSize
+- SetCalUserDefinedInfo
+- GetCalUserDefinedInfo
+- InitExtCal
```
Comment (kruthi-NI) [inline source/codegen/metadata/nidcpower/CHANGES.md:66]: Remove other doc in this file related to InitExtCal and CloseExtCal before submitting

Comment (ni-pkhare) [reply source/codegen/metadata/nidcpower/CHANGES.md:66]: Done

File: source/codegen/metadata/nidcpower/CHANGES.md [hunk 3]
```diff
@@ -54,6 +52,29 @@ Following enums were defined :
     - NIDCPOWER_VAL_POWER_ALLOCATION_MODE_AUTOMATIC to NIDCPOWER_VAL_AUTOMATIC
     - NIDCPOWER_VAL_POWER_ALLOCATION_MODE_MANUAL to NIDCPOWER_VAL_MANUAL
 
+The following enums were removed :
```
Comment (doshirohan) [inline source/codegen/metadata/nidcpower/CHANGES.md:55]: By looking at the lines removed above, seems like these enums were not part of nimi-python metadata. These were added by us. So just removing lines above should be sufficient, we don't need to document these as "removed enums".

File: source/codegen/metadata/nidcpower/CHANGES.md [hunk 4]
```diff
@@ -54,6 +52,29 @@ Following enums were defined :
...
+- InternalReference
+- CalibrationCloseAction
+
+## functions.py
+
+Following calibration functions were removed from the metadata :
```
Comment (doshirohan) [inline source/codegen/metadata/nidcpower/CHANGES.md:61]: These removed functions should be documented in the `functions.py` section below.

**Review by doshirohan — APPROVED** [review]

File: source/codegen/metadata/nidcpower/CHANGES.md (hunk 4 above)
Comment (ni-pkhare) [reply source/codegen/metadata/nidcpower/CHANGES.md]: Done.

File: source/codegen/metadata/nidcpower/CHANGES.md (hunk 3 above)
Comment (ni-pkhare) [reply source/codegen/metadata/nidcpower/CHANGES.md]: Done.

**Pull request merged**